composer.json

```diff
@@ -14,13 +14,24 @@
     }
   ],
   "autoload": {
-    "psr-0": {
-      "PAMI": "src/mg"
+    "psr-4": {
+      "PAMI\\": "src/mg/PAMI"
     }
   },
   "require": {
     "php": ">=5.3.3",
     "apache/log4php": "2.3.0",
     "psr/log": "~1.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "4.8.*",
+    "phing/phing": "2.*",
+    "squizlabs/php_codesniffer": "2.*",
+    "sebastian/phpcpd": "*",
```

Comment:
This ain't nice :smile:. It basically means: give me the latest version of this packages, whatever it is. Might cause major breaks when someone releases a new major version.

Quick tip: removing this line and then peforming `composer.phar require <package>` will automatically choose the latest version _and_ set the correct require line for a package.

Same goes for the other includes.